system-addon/test/unit/lib/TelemetrySender.test.js

```diff
@@ -122,87 +96,49 @@ describe("TelemetrySender", () => {
     it("should log HTTP failures using Cu.reportError", async () => {
       fetchStub.resolves(fakeFetchHttpErrorResponse);
 
-      await tSender._sendPing(fakePingJSON);
+      await tSender.sendPing(fakePingJSON);
 
       assert.called(Components.utils.reportError);
     });
 
     it("should log an error using Cu.reportError if fetch rejects", async () => {
       fetchStub.rejects("Oh noes!");
 
-      await tSender._sendPing(fakePingJSON);
+      await tSender.sendPing(fakePingJSON);
 
       assert.called(Components.utils.reportError);
     });
 
     it("should log if logging is on && if action is not activity_stream_performance", async () => {
-      FakePrefs.prototype.prefs = {
-        "telemetry": true,
-        "performance.log": true
-      };
+      FakePrefs.prototype.prefs = {};
+      FakePrefs.prototype.prefs[TELEMETRY_PREF] = true;
+      FakePrefs.prototype.prefs[LOGGING_PREF] =  true;
       fetchStub.resolves(fakeFetchSuccessResponse);
       tSender = new TelemetrySender(tsArgs);
 
-      await tSender._sendPing(fakePingJSON);
-
-      assert.called(console.log); // eslint-disable-line no-console
-    });
-  });
-
-  describe("#observe()", () => {
-    before(() => {
-      globals.sandbox.stub(TelemetrySender.prototype, "_sendPing");
-    });
-
-    observerTopics.forEach(topic => {
-      it(`should call this._sendPing with data for ${topic}`, () => {
-        const fakeSubject = "fakeSubject";
-        tSender = new TelemetrySender(tsArgs);
-
-        tSender.observe(fakeSubject, topic, fakePingJSON);
-
-        assert.calledOnce(TelemetrySender.prototype._sendPing);
-        assert.calledWithExactly(TelemetrySender.prototype._sendPing,
-          fakePingJSON);
-      });
-    });
-
-    it("should not call this._sendPing for 'nonexistent-topic'", () => {
-      const fakeSubject = "fakeSubject";
-      tSender = new TelemetrySender(tsArgs);
-
-      tSender.observe(fakeSubject, "nonexistent-topic", fakePingJSON);
+      await tSender.sendPing(fakePingJSON);
 
-      assert.notCalled(TelemetrySender.prototype._sendPing);
+      assert.called(global.dump);
```

Comment:
Whoops, I didn't even see the Console.jsm import! I didn't even know this existed... ha!

In that case, do you mind if I just change the global override of `console`? I was stumped for quite a while trying to figure out why console wasn't working in test debugging...